packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -228,6 +231,15 @@ export const Table = memo((props: Props) => {
     setPageSize(pageSize);
   }, [pageSize, setPageSize]);
 
+  let scrollTop = undefined;
+  if (rowHighlightIndex !== undefined) {
+    const firstMatchedRowIndex = rows.findIndex((row) => row.index === rowHighlightIndex);
```

Comment:
I know findIndex returns the first instance of the match, but for the sake of curiosity, would there ever be more then 1 unique time value in a table?

Reply:
There *could* be, but I don't think there are many use-cases for such a scenario. I've also thought about it and decided it's better to return just one match.